gravatar-ui/api/gravatar-ui.api

```diff
@@ -146,34 +151,45 @@ public final class com/gravatar/ui/components/ComposableSingletons$ProfileSummar
 public final class com/gravatar/ui/components/LargeProfileKt {
 	public static final fun DisplayNamePreview (Landroidx/compose/runtime/Composer;I)V
 	public static final fun LargeProfile (Lcom/gravatar/api/models/Profile;Landroidx/compose/ui/Modifier;Landroidx/compose/runtime/Composer;II)V
-	public static final fun LargeProfile (Lcom/gravatar/ui/components/ComponentState;Landroidx/compose/ui/Modifier;Lkotlin/jvm/functions/Function3;Lkotlin/jvm/functions/Function3;Landroidx/compose/runtime/Composer;II)V
+	public static final fun LargeProfile (Lcom/gravatar/restapi/models/Profile;Landroidx/compose/ui/Modifier;Landroidx/compose/runtime/Composer;II)V
+	public static final fun LargeProfile (Lcom/gravatar/ui/components/ComponentState;Landroidx/compose/ui/Modifier;Landroidx/compose/runtime/Composer;II)V
+	public static final fun LargeProfileWithComponentState (Lcom/gravatar/ui/components/ComponentState;Landroidx/compose/ui/Modifier;Lkotlin/jvm/functions/Function3;Lkotlin/jvm/functions/Function3;Landroidx/compose/runtime/Composer;II)V
 }
 
 public final class com/gravatar/ui/components/LargeProfileSummaryKt {
 	public static final fun LargeProfileLoadingPreview (Landroidx/compose/runtime/Composer;I)V
 	public static final fun LargeProfileSummary (Lcom/gravatar/api/models/Profile;Landroidx/compose/ui/Modifier;Landroidx/compose/runtime/Composer;II)V
-	public static final fun LargeProfileSummary (Lcom/gravatar/ui/components/ComponentState;Landroidx/compose/ui/Modifier;Lkotlin/jvm/functions/Function3;Lkotlin/jvm/functions/Function3;Landroidx/compose/runtime/Composer;II)V
```

Comment:
This will be the one not released yet, right?

Reply:
Yes :)